src/components/common/form/SelectGroup.js

```diff
@@ -95,7 +95,7 @@ const SelectGroup = props => {
 				{content}
 			</Select>
 
-			<FormHelperText id={`${name}-error-text`}>{error}</FormHelperText>
+			{error ? <FormHelperText id={`${name}-error-text`}>{error}</FormHelperText> :  null}
```

Comment:
You can use the short form of this when there is only one option.
```suggestion
			{error && <FormHelperText id={`${name}-error-text`}>{error}</FormHelperText>}
```